client/src/components/ItemSummary.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import $ from 'jquery';
+import NameAndPrice from './NameAndPrice.jsx';
+import Description from './Description.jsx';
+
+class ItemSummary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      item: [],
+    };
+    this.getFirstItem = this.getFirstItem.bind(this);
+  }
+
+  componentDidMount() {
+    const id = window.location.pathname.substring(1)
+    this.getItemById(id);
+  }
+
+  getItemById(id) {
+    $.ajax({
```

Comment:
Might be worth considering more efficient libraries for API calls like axios or fetch instead of jQuery

Reply:
I'm not 100% familiar with those libraries yet, but I think this might be worth a refactor. I've read that $.ajax is good for accessibility (I guess it's one of the few things IE actually supports)